src/util/GeoSparqlHelpers.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright 2022, University of Freiburg,
+// Author: Hannah Bast <bast@cs.uni-freiburg.de>
+// Chair of Algorithms and Data Structures
+
+#include "./GeoSparqlHelpers.h"
+
+#include <absl/strings/charconv.h>
+#include <ctre/ctre.h>
+
+#include <cmath>
+#include <limits>
+#include <numbers>
+#include <optional>
+#include <string>
+#include <string_view>
+
+#include "./Exception.h"
+
+namespace ad_utility {
+
+namespace detail {
+
+static constexpr auto wktPointRegex = ctll::fixed_string(
+    "^\"[Pp][Oo][Ii][Nn][Tt]\\s*\\( *"
+    "(-?[0-9]+|-?[0-9]+\\.[0-9]+)"
+    "\\s+"
+    "(-?[0-9+]|-?[0-9]+\\.[0-9]+)"
+    "\\s*\\)\"(\\^\\^|$)");
```

Comment:
Technically there still is a `space*` after the opening bracket, allowing POINT( 1... but not POINT(\t3....). But when parsing WKT properly we will have to consider the standard documents more in detail. I think you are allowed arbitrary whitespace between all the different "elements" here, but this is not super important, so if you are ok with this, you can merge it:)